pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -163,6 +163,6 @@ func setupConntrackParameters() error {
 }
 
 func (ct *connTrackSystem) GetMaxConnections() (int, error) {
-	maxConns, err := sysctl.GetSysctlNet("nf_conntrack_max")
+	maxConns, err := sysctl.GetSysctlNet("netfilter/nf_conntrack_max")
```

Comment:
was that broken before, i.e. was the Prometheus metric not updated correctly? If yes, it should be mentioned in the final commit message / PR description

Reply:
I have done the test and it works well.

Reply:
we clarified offline: "nf_conntrack_max" and "netfilter/nf_conntrack_max" are supposed to be the same, but on the HyperKit Linux VM used for docker containers on macOS, "nf_conntrack_max" is not recognized. Thanks @dreamtalen